### DISCUSSION TITLE
fake NAT-T enabled always

### DISCUSSION THREAD

**Discussion by antonymsouza** [strongswan/strongswan#2549] [General]
Hi,

We have set mobike=no, forceencaps=no. As we are running strongswan 5.7.2rc1+vpp+dpdk in the user space, disabled kernel-libipsec, kernel-netlink plugins from the strongswan.conf.

Still we can see the fake NAT-T enabled message in the logs and IKE AUTH and messages after as well as ESP packets are encapsulated over UDP 4500.

Thanks,
Antony

**Comment by tobiasbrunner** [comment]
> As we are running strongswan 5.7.2rc1+vpp+dpdk

I've no idea what that version does, so I can't help you, sorry.

**Reply by antonymsouza** [reply]
No problem, we will check with latest Strongswan and let you know if the issue persists.
